data/json/monsters/zed_misc.json

```diff
@@ -89,6 +89,59 @@
     },
     "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "STUMBLES", "WARM", "POISON", "GRABS", "NO_BREATHE", "FILTHY" ]
   },
+  {
+    "id": "mon_zombullfrog",
+    "type": "MONSTER",
+    "name": { "str": "zombullfrog" },
+    "description": "The wound covered skin of this large bullfrog appears to be peeling in places, revealing a black goop beneath the surface.  Its vacant stare passes by anything that isn't food, and it will leap to devour anything it can.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "120 L",
+    "weight": "150 kg",
+    "hp": 120,
+    "speed": 50,
+    "material": [ "flesh" ],
+    "symbol": "F",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "bash", "amount": 5 } ],
+    "dodge": 1,
+    "armor_bash": 8,
+    "armor_cut": 2,
+    "armor_bullet": 1,
+    "armor_elec": 2,
+    "bleed_rate": 0,
+    "vision_night": 7,
+    "harvest": "zombie",
+    "special_attacks": [
+      { "type": "leap", "cooldown": 12, "move_cost": 0, "max_range": 4, "min_consider_range": 2 },
+      [ "RANGED_PULL", 10 ],
+      [ "GRAB_DRAG", 3 ]
+    ],
+    "regen_morale": true,
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "KEENNOSE",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "GRABS",
```

Comment:
You don't need GRABS if it has a special attack, and you don't need FILTHY if they don't drop gear. You can also leave POISON out if their harves list is set up for tainted meat.

Reply:
Gone. I had simply just copy and pasted the frog mother entry, so those flags are all on the frog mother as well.